### PR TITLE
Adding blog post "Creating Patch Files for WildFly"

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: posts/Gunnar/2017-05-29-creating-patches-for-wildfly.adoc [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+            </goals>
+            <configuration>
+                <artifactItems>
+                    <artifactItem>
+                        <groupId>org.wildfly</groupId>
+                        <artifactId>wildfly-dist</artifactId>
+                        <version>${wildfly.version}</version>
+                        <type>tar.gz</type>
+                        <overWrite>false</overWrite>
+                        <outputDirectory>${project.build.directory}/wildfly-original</outputDirectory>
```
Comment (gsmet) [inline posts/Gunnar/2017-05-29-creating-patches-for-wildfly.adoc:113]: Looks like you could use your `${wildflyOriginal}` and `${wildflyPatched}` variables here.

**Review by gsmet — COMMENTED** [review]
Looks very good.

I added a very minor comment.

File: posts/Gunnar/2017-05-29-creating-patches-for-wildfly.adoc (hunk 1 above)
Comment (gunnarmorling) [reply posts/Gunnar/2017-05-29-creating-patches-for-wildfly.adoc:113]: Those vars contain the version at the end: 

    <wildflyOriginal>${project.build.directory}/wildfly-original/wildfly-${wildfly.version}</wildflyOriginal>
    <wildflyPatched>${project.build.directory}/wildfly-patched/wildfly-${wildfly.version}</wildflyPatched>

I guess one could strip that top-level dir when unpacking somehow, but it doesn't seem worth the extra effort.

**Comment by gunnarmorling** [comment]
Thanks for reviewing, @gsmet!

**Pull request merged**